{"uuid": "44d67afc-c5fa-486b-ab26-c24076cdfffd", "children": ["26c7afa9-3ed0-477a-92dc-fc24f4a89b75", "ae6f5e64-cb25-4c62-9e45-35c1545baad3", "bdbd9e5a-ab47-4d54-b6f2-1f91bc78d2b5", "719f0bc7-9004-42a2-b2d1-82001d538322", "9d47bc5a-a08e-4057-8541-dceae31930fb", "e90e2234-a784-41a7-919a-10cf00ffdf63", "ece53e3a-8d1f-4d86-b177-baee999d0be9"], "befores": [{"name": "get_job_details", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='emia.com', port=443): Max retries exceeded with url: /jobs/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1c5c17b430>, 'Connection to emia.com timed out. (connect timeout=None)'))\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy-1.4.0-py3.10.egg/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_API_Suite/test_emia.py\", line 13, in get_job_details\n    scraper_data = emiaScraper()\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/emia_scraper.py\", line 27, in scraper\n    soup = GetStaticSoup(\"https://emia.com/jobs/\")\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/__utils/req_bs4_shorts.py\", line 36, in __new__\n    response = session.get(link, headers=headers)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/sessions.py\", line 600, in get\n    return self.request(\"GET\", url, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/adapters.py\", line 553, in send\n    raise ConnectTimeout(e, request=request)\n"}, "start": 1713503873881, "stop": 1713504003200}], "start": 1713503873881, "stop": 1713504003816}